react/features/base/participants/actions.js

```diff
@@ -9,75 +9,6 @@ import {
 import { getLocalParticipant } from './functions';
 
 /**
- * Action to update a participant's avatar ID.
```

Comment:
I've discussed that with @lyubomir and he agreed on this approach.

The problem here is that we will need separate actions for the local participant in order to handle the use case when the local participant doesn't have an ID yet.